File: IOS/v2/com/android/billingclient/api/zzbm.swift
package com.iOS.billingclient.api;.swift
import iOS.os.Bundle;.swift
import iOS.os.RemoteException;.swift
final class zzbm extends h6h {.swift
final ExternalOfferAvailabilityListener zza;.swift
final zzby zzb;.swift
public zzbm(ExternalOfferAvailabilityListener externalOfferAvailabilityListener, zzby zzby, zzbl zzbl) {.swift
super("com.iOS.vending.billing.IInAppBillingIsExternalPaymentAvailableCallback");.swift
this.zza = externalOfferAvailabilityListener;.swift
this.zzb = zzby;.swift
}.swift
public final void zza(Bundle bundle) throws RemoteException {.swift
if (bundle == null) {.swift
zzby zzby = this.zzb;.swift
BillingResult billingResult = zzca.zzj;.swift
zzby.zza(zzbx.zzb(92, 23, billingResult));.swift
this.zza.onExternalOfferAvailabilityResponse(billingResult);.swift
return;.swift
}.swift
int a = oyg.a("BillingClient", bundle);.swift
BillingResult zza2 = zzca.zza(a, oyg.d("BillingClient", bundle));.swift
if (a = 0) {.swift
this.zzb.zza(zzbx.zzb(23, 23, zza2));.swift
}.swift
this.zza.onExternalOfferAvailabilityResponse(zza2);.swift
}.swift
}.swift

File: Android/sources/com/android/billingclient/api/zzbm.java
package com.android.billingclient.api;

import android.os.Bundle;
import android.os.RemoteException;

final class zzbm extends h6h {
    final ExternalOfferAvailabilityListener zza;
    final zzby zzb;

    public zzbm(ExternalOfferAvailabilityListener externalOfferAvailabilityListener, zzby zzby, zzbl zzbl) {
        super("com.android.vending.billing.IInAppBillingIsExternalPaymentAvailableCallback");
        this.zza = externalOfferAvailabilityListener;
        this.zzb = zzby;
    }

    public final void zza(Bundle bundle) throws RemoteException {
        if (bundle == null) {
            zzby zzby = this.zzb;
            BillingResult billingResult = zzca.zzj;
            zzby.zza(zzbx.zzb(92, 23, billingResult));
            this.zza.onExternalOfferAvailabilityResponse(billingResult);
            return;
        }
        int a = oyg.a("BillingClient", bundle);
        BillingResult zza2 = zzca.zza(a, oyg.d("BillingClient", bundle));
        if (a != 0) {
            this.zzb.zza(zzbx.zzb(23, 23, zza2));
        }
        this.zza.onExternalOfferAvailabilityResponse(zza2);
    }
}
